bin/deseq2.R

```diff
@@ -534,8 +534,8 @@ for (comparison in comparisons) {
   ##########################################
 
   factor <- "condition"
-  numerator <- l2
-  denominator <- l1
+  numerator <- l1
+  denominator <- l2
```

Comment:
also here @MarieLataretu , I just want to be sure we don't revert something important :) Is this the current correct and expected behavior?

Reply:
@fischer-hub if this is confirmed, please feel free to merge this PR!

Reply:
This is correct. We want the current master content: https://github.com/hoelzer-lab/rnaflow/blob/master/bin/deseq2.R#L537-L538